proposal.rst

```diff
@@ -162,7 +161,7 @@ FILLME
 .. these are few, it is usually possible to get enough done across Summer of
 .. Code to make for a worthwhile project.
 
-FILLME
+Monday–Friday 7am-1am UTC, but I can move it if needed.
```

Comment:
You seem to be suggesting you'll work 18 hours a day, for a total of 90 hours a week. I'd gently suggest that this is far too much to contribute at your best capability for a sustained project across months.